lib/openid_connect/authorization_point/endpoint_start_point.rb

```diff
@@ -0,0 +1,11 @@
+module OpenidConnect
+  module AuthorizationPoint
+    class EndpointStartPoint < Endpoint
+      def handle_response_type(req, res)
```

Comment:
Unused method argument - `res`. If it's necessary, use `_` or `_res` as an argument name to indicate that it won't be used.